tokenizer/src/lib.rs

```diff
@@ -201,13 +205,13 @@ impl<'a> TokenStream<'a> {
                         let (offset0, cp0) = chars.next().unwrap();
 
                         if cp0.is_id_start() {
-                            self.index += offset0 as u32 + 1;
+                            self.index = start + offset0 as u32;
```

Comment:
Isn't `offset0` always 0, since it's the offset at which the first Unicode character starts in the slice?

Reply:
Oh, yeah could not wrap this section in my head it seems.  Thanks